domains/primitives/runtime/src/lib.rs

```diff
@@ -31,15 +31,15 @@ use frame_system::limits::{BlockLength, BlockWeights};
 use parity_scale_codec::{Decode, Encode};
 use scale_info::TypeInfo;
 use serde::{Deserialize, Serialize};
-use sp_runtime::generic::UncheckedExtrinsic;
-use sp_runtime::traits::{Convert, IdentifyAccount, Verify};
+use sp_runtime::generic::{Preamble, UncheckedExtrinsic};
+use sp_runtime::traits::transaction_extension::TransactionExtension;
+use sp_runtime::traits::{Convert, Dispatchable, IdentifyAccount, Verify};
 use sp_runtime::transaction_validity::TransactionValidityError;
 use sp_runtime::{MultiAddress, MultiSignature, Perbill};
 use sp_weights::constants::WEIGHT_REF_TIME_PER_SECOND;
 use sp_weights::Weight;
 pub use subspace_runtime_primitives::HoldIdentifier;
 use subspace_runtime_primitives::{MAX_BLOCK_LENGTH, SHANNON};
-
```

Comment:
Nit: looks like removing this line was a mistake, we usually separate imports from types.